Sources/Error.swift

```diff
@@ -1,8 +1,8 @@
-public protocol Error: ErrorProtocol {
+public protocol S4Error: Error {
```

Comment:
You think? I dislike having that symbol conflict, that's why I prefer just renaming it to something like `S4Error`. I know we can disambiguate it like this, I just don't like the lack of autocomplete etc.

What do other people think?